ceremonies/rpc/src/lib.rs

```diff
@@ -38,26 +45,87 @@ where
 	) -> Result<Vec<(CeremonyIndexType, CommunityReputation)>>;
 }
 
-pub struct Ceremonies<Client, Block, AccountId> {
+pub struct Ceremonies<Client, Block, AccountId, S> {
 	client: Arc<Client>,
 	_marker: std::marker::PhantomData<(Block, AccountId)>,
```

Comment:
Minor detail (I know this has been like this before). The `_marker` is usually the last field because it is never used.

Reply:
i dont understand, please elaborate